go/vt/vttablet/tabletserver/vstreamer/vstreamer_test.go

```diff
@@ -1487,7 +1560,7 @@ func TestBestEffortNameInFieldEvent(t *testing.T) {
 
 // todo: migrate to new framework
 // test that vstreamer ignores tables created by OnlineDDL
-func TestInternalTables(t *testing.T) {
+func TestOnlineDDLTables(t *testing.T) {
```

Comment:
This is a clue that it might be better to explicitly call these `sidecar[db]_tables` rather than `internal_tables`. Although they're all internal tables so it's also fine as-is IMO.

Reply:
yeah, I was also vacillating between `internal` and `sidecar(db)`. `internal` is not precise because we could also term online ddl shadow tables and other artifacts as internal. I will let @deepthi also weigh in before changing it everywhere.

Reply:
This seems to have been resolved.

Reply:
@deepthi, the question was a broader one. Currently we use options/proto attributes like `InternalTables`, `is_internal`. Should we change that to  `SidecarDBTables` and `is_sidecardb_table`?

We could either continue calling it Internal (keeping it generic, allowing for future specification of other tables than SidecarDB tables, like, I don't know, shadow tables in OnlineDDL for example) or lock it down to SidecarDB tables using that in the object naming instead.

Reply:
`internal` is fine for now. Renaming proto fields is not difficult if we wanted to change it in the future.